### PR TITLE
⭐️ powerpc 64 little-endian build

### PR DESCRIPTION
**Pull request by chris-rock**
As per user request, we are adding power pc architecture to our build pipeline.

### CODE REVIEW

**Comment by tas50** [comment]
Let's make sure to update the docs with this support.

File: .goreleaser.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ builds:
       - 386
       - arm64
       - arm
+      - ppc64le
```
Comment (czunker) [inline .goreleaser.yml:16]: Do we also need a docker image for power?

Comment (chris-rock) [reply .goreleaser.yml:16]: I think we should, but I wanted to split this in iterations. First lets test binary build and then we can test the docker build.

**Review by arlimus — APPROVED** [review]
sweet addition!!

**Pull request merged**